src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1577,11 +1577,11 @@ def _get(self, entity, entnum, item1, it1num, item2, it2num):
             self._get_lock = False
 
         if getresponse.type == 0:
-            raise ValueError(
-                "This is either an invalid get request, or MAPDL is set"
-                " to the wrong processor (e.g. on BEGIN LEVEL vs."
-                " POST26)"
-            )
+            # Fall back to run to get most verbose output.
+            out = self.run("*GET,__temp__," + cmd)
+            self.run("__temp__=") # deleting parameter
```

Comment:
```suggestion
            self.run("__temp__=", mute=False)  # deleting parameter
```

Reply:
Why to do you need to specify ``mute=False`` for deleting a parameter?